packages/create-sitecore-jss/src/templates/nextjs/src/pages/api/revalidate.ts

```diff
@@ -0,0 +1,30 @@
+﻿import { RevalidateMiddleware } from '@sitecore-jss/sitecore-jss-nextjs/revalidate';
+import { NextApiResponse, NextApiRequest } from 'next';
+import clientFactory from 'lib/graphql-client-factory';
+
+const handler = async (req: NextApiRequest, res: NextApiResponse) => {
```

Comment:
I'm thinking if we need to add a description, so the purpose of this api route will be clear for the customer